src/components/FooterFolder/styles.ts

```diff
@@ -1,27 +1,159 @@
 import styled from 'styled-components';
+import { ArrowUpRight, Facebook, Instagram, Youtube } from 'react-feather';
 import COLORS from '../../styles/colors';
 
+export const Heading = styled.h1`
+  font-family: 'Public Sans', sans-serif;
+  color: black;
+  font-size: 40px;
+`;
+
 export const ContactDiv = styled.div`
-  width: 250px;
-  height: 250px;
-  margin-right: 130px;
-  margin-left: 30px;
-  color: ${COLORS.black};
-  text-align: center;
-  font-family: Public Sans;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  marginright: 90px;
+  marginleft: 90px;
+  gap: 18px;
+`;
+
+export const LinkDiv = styled.div`
+  display: flex;
+  align-items: flex-start;
+  gap: 60px;
+  marginright: 90px;
```

Comment:
Be sure to have a dash between the words here! Should be `margin-right` and `margin-left`.

Alternatively, we can also use the margin shorthand for this! Rather than two separate properties for left and right, you can specify one like so:

`margin: <top-bottom-values> <left-right-values>`

So in this case, you would use: `margin: 0 90px`